bindings/python/iota_sdk/types/transaction_with_metadata.py

```diff
@@ -23,12 +23,12 @@ class InclusionState(str, Enum):
         Conflicting: The transaction is conflicting.
         UnknownPruned: The transaction is unknown or already pruned.
     """
-    Pending = 'pending'
-    Accepted = 'accepted'
-    Confirmed = 'confirmed'
-    Finalized = 'finalized'
-    Conflicting = 'conflicting'
-    UnknownPruned = 'unknownPruned'
+    Pending = 'Pending'
```

Comment:
Why?

Reply:
Serialization for this type is like this

Reply:
But it's it supposed to be camelCase?

Reply:
I think this is not a TIP type so we don't really care?

Reply:
okie dokie 🤷‍♀️